Expecto.Tests/Tests.fs

```diff
@@ -531,7 +537,7 @@ let expecto =
         All elements in `expected` ['item', 'number of expected occurrences']:
         {2: 1, 4: 1}
         Missing elements from `actual`:
-        
+
```

Comment:
@haf, this is a change that makes test fail. Does we really should depend on having empty lines containing spaces in error messages?